scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -45,6 +45,13 @@ public class CompileTargets
             "Microsoft.Extensions.Testing.Abstractions"
         };
 
+        // Updates the stage 2 with recent changes.
+        [Target(nameof(PrepareTargets.Init), nameof(CompileStage2))]
+        public static BuildTargetResult UpdateBuild(BuildTargetContext c)
```

Comment:
It even skips stage1, which puts your artifacts folder in a weird state, but that's just fine for this :)